release-notes/aws/v11.2.0.md

```diff
@@ -1,8 +1,16 @@
-# :zap: Giant Swarm Release 11.1.1 for AWS is now active for you! :zap:
+# AWS v11.2.0
 
-## aws-operator 8.2.1
+__Note:__ Upgrading to this release from any release prior 11.1.1 will cause a network downtime due to the network-related changes coming with the switch from Calico CNI to AWS CNI.
 
-- Replaces Calico CNI (Container Network Interface) with [AWS CNI](https://github.com/aws/amazon-vpc-cni-k8s) and enables Pod to Pod networking throughout AWS regions. As a result of this change, CIDR (IPv4 address blocks) for tenant cluster Pods are changing. They now must adhere to the rules described in the [AWS documentation](https://docs.aws.amazon.com/vpc/latest/userguide/VPC_Subnets.html#add-cidr-block-restrictions).
+## aws-operator 8.2.2
```

Comment:
We usually have the `v` prefix to versions here. Linking to the `CHANGELOG.md` in the operator can come once it's finished.

Reply:
Changed